docs/CONTRIBUTING.md

```diff
@@ -66,11 +95,11 @@ ChangeLog: Foo plugin: A specific issue people had has been fixed.
 ```
 
 The summary must be on a line of its own, with a "ChangeLog:" prefix at the
-beginning of the line. The text should start with "Foo plugin" to give the
-reader context for the information. Other common contexts are "collectd" for
-the core daemon, "Build system", and "Documentation". Use past tense and
-passive voice the for remainder, e.g. "a bug has been fixed", "a feature has
-been added".
+beginning of the line. The text should start with "Foo plugin" or "collectd"
+(when making changes to the daemon itself) to give the reader context for the
+information. Other common contexts are "collectd" for the core daemon, "Build
+system", and "Documentation". Use past tense and passive voice the for
```

Comment:
"collectd" context is mentioned twice, maybe:
```suggestion
beginning of the line. To give the reader context for the information, the text
should start with "<name of> plugin" when making changes to a plugin, and
"collectd" for the core daemon changes.  Other common contexts are "Build
system" and "Documentation". Use past tense and passive voice the for
```

Reply:
Great, thanks!